plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -61,15 +62,18 @@ public HiveMinioDataLake(String bucketName, Map<String, String> hiveHadoopFilesT
                                 .put("MINIO_SECRET_KEY", SECRET_KEY)
                                 .buildOrThrow())
                         .build());
-        this.hiveHadoop = closer.register(
-                HiveHadoop.builder()
-                        .withFilesToMount(ImmutableMap.<String, String>builder()
-                                .put("hive_minio_datalake/hive-core-site.xml", "/etc/hadoop/conf/core-site.xml")
-                                .putAll(hiveHadoopFilesToMount)
-                                .buildOrThrow())
-                        .withImage(hiveHadoopImage)
-                        .withNetwork(network)
-                        .build());
+
+        HiveHadoop.Builder hiveHadoopBuilder = HiveHadoop.builder()
+                .withImage(hiveHadoopImage)
+                .withNetwork(network);
+        if (hiveHadoopFilesToMount.isEmpty()) {
```

Comment:
Not sure here but shouldn't we always copy resources "hive_minio_datalake/hive-core-site.xml"
and if there are any files to mount then also files like
```
HiveHadoop.Builder hiveHadoopBuilder = HiveHadoop.builder()
                .withImage(hiveHadoopImage)
                .withNetwork(network)
                .withResourcesToMount(ImmutableMap.of("hive_minio_datalake/hive-core-site.xml", "/etc/hadoop/conf/core-site.xml"));
if (!hiveHadoopFilesToMount.isEmpty()) {
    hiveHadoopBuilder.withFilesToMount(hiveHadoopFilesToMount);
}
```
We could potentially just blindly allways add withFilesToMount. It it's empty forEach will not iterate and do not mount them